README.md

```diff
@@ -38,15 +41,22 @@ To get a local copy up and running follow these simple example steps.
 
 ## 🤝 Contributing
 
+
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
+## Acknowledgements
+
+This project is inspired to: [ Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template CC-Global-Summit   Cindy Shin, I'm so gratfull for all.
+
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is inspired to **Cindy Shin in Behance** template CAMPANY STUFF_Conferance
+This project is not licensed.
```

Comment:
[OPTIONAL] Why your project is not licensed? I think it would be better to have the MIT license in it.

Reply:
Okay thank you let me work on it then